superset/views/core.py

```diff
@@ -361,15 +362,16 @@ def clean_fulfilled_requests(session: Session) -> None:
         else:
             flash(__("You have no permission to approve this request"), "danger")
             return redirect("/accessrequestsmodelview/list/")
-        for r in requests:
-            session.delete(r)
+        for request_ in requests:
+            session.delete(request_)
```

Comment:
I think `request` is a viable non-conflicting name.

Reply:
`request` is imported from `flask` in the outer scope.